pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -321,5 +337,22 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 		"portMappings": portMappingsParam,
 	}
 
+	ingress, egress, err := bandwidth.ExtractPodBandwidthResources(annotations)
+	if err != nil {
+		return nil, fmt.Errorf("Error reading pod bandwidth annotations: %v", err)
+	}
+	if ingress != nil || egress != nil {
+		bandwidthParam := cniBandwidthEntry{}
+		if ingress != nil {
+			bandwidthParam.IngressRate = int(ingress.Value() / 1000)
```

Comment:
The documentation was wrong; the units are bits per second.

Reply:
Done. Thanks!